packages/salesforce/src/Adaptor.js

```diff
@@ -114,12 +131,16 @@ export function describe(sObject) {
 /**
  * Retrieves a Salesforce sObject(s).
  * @public
- * @example
- * retrieve('ContentVersion', '0684K0000020Au7QAE/VersionData');
+ * @example <caption>Retrieve a specific ContentVersion record</caption>
+ * retrieve('ContentVersion', '0684K0000020Au7QAE', state => {
+ *   console.log(state.references[0]); // Log the retrieved record
```

Comment:
Is the comment here useful? `console.log()` is  a pretty easy to understand function